controller/src/beerocks/master/son_master_thread.h

```diff
@@ -54,6 +54,9 @@ class master_thread : public beerocks::socket_thread {
     void handle_cmdu_control_ieee1905_1_message(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx);
     bool handle_intel_slave_join(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx,
                                  ieee1905_1::CmduMessageTx &cmdu_tx);
+    bool handle_non_intel_slave_join(Socket *sd, std::shared_ptr<ieee1905_1::tlvWscM1> tlvwscM1,
+                                     std::string bridge_mac, std::string radio_mac,
```

Comment:
Better to use `const std::string` or `std::string &` to avoid copy constructor since we're not changing `bridge_mac` and `radio_mac`.